File: Biblioteca/src/servicios/BibliotecaInterfaz.java
package servicios;

import java.util.List;

import dtos.ClienteDto;
import dtos.bibliotecaDto;

public interface BibliotecaInterfaz {

	public void bibliotecaNueva(List<bibliotecaDto> listaAntigua);
	
	public void bibliotecaExistente(List<bibliotecaDto> listaAntigua, List<ClienteDto> listaClientes);
}

File: Ejercicio1Global/servicios/BibliotecaInterfaz.cs
using Ejercicio1Global.Dtos;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejercicio1Global.servicios
{
    internal interface BibliotecaInterfaz
    {
        public void bibliotecaNueva(List<Biblioteca> listaAntigua);

        public void bibliotecaExistente(List<Biblioteca> listaAntigua, List<ClientesDto> listaClientes, List<LibrosDto> librosDtos);
    }
}
